doc/source/gotcha.rst

```diff
@@ -1,20 +1,94 @@
 .. currentmodule:: pandas
-.. _gotchas:
+.. _faq:
```

Comment:
Ah, but I see you changed the title below, then the link name change makes some sense